v03_pipeline/lib/annotations/mito.py

```diff
@@ -0,0 +1,75 @@
+from __future__ import annotations
+
+from typing import Any
+
+import hail as hl
+
+from v03_pipeline.lib.annotations.enums import MITOTIP_PATHOGENICITIES
+
+MITOTIP_PATHOGENICITIES_LOOKUP = hl.dict(
+    hl.enumerate(MITOTIP_PATHOGENICITIES, index_first=False).extend(
+        # NB: adding missing values here allows us to
+        # hard fail if a mapped key is present and has an unexpected value
+        # but propagate missing values.
+        [(hl.missing(hl.tstr), hl.missing(hl.tint32))],
+    ),
+)
+
+
+def common_low_heteroplasmy(ht: hl.Table, **_: Any) -> hl.Expression:
+    return ht.common_low_heteroplasmy
+
+
+def callset_heteroplasmy(ht: hl.Table, **_: Any) -> hl.Expression:
+    return hl.Struct(
+        AF=ht.AF_het,
+        AC=ht.AC_het,
+        AN=ht.AN,
+    )
+
+
+def contamination(mt: hl.MatrixTable, **_: Any) -> hl.Expression:
+    return mt.contamination
+
+
+def DP(mt: hl.MatrixTable, **_: Any) -> hl.Expression:  # noqa: N802
+    is_called = hl.is_defined(mt.GT)
+    return (hl.cond(is_called, hl.int(hl.min(mt.DP, 32000)), hl.missing(hl.tfloat)),)
+
+
+def GQ(mt: hl.MatrixTable, **_: Any) -> hl.Expression:  # noqa: N802
+    is_called = hl.is_defined(mt.GT)
+    return hl.if_else(is_called, mt.MQ, 0)
+
+
+def haplogroup(ht: hl.Table, **_: Any) -> hl.Expression:
+    return hl.Struct(
+        is_defining=hl.if_else(ht.hap_defining_variant, 0, hl.missing(hl.tint)),
+    )
+
+
+def HL(mt: hl.MatrixTable, **_: Any) -> hl.Expression:  # noqa: N802
+    is_called = hl.is_defined(mt.GT)
+    return hl.if_else(is_called, mt.HL, 0)
+
+
+def high_constraint_region(
+    ht: hl.Table,
+    interval_mito_ht: hl.Table,
+    **_: Any,
+) -> hl.Expression:
+    return hl.is_defined(interval_mito_ht[ht.locus])
+
+
+def mito_cn(mt: hl.MatrixTable, **_: Any) -> hl.Expression:
+    return hl.int(mt.mito_cn)
+
+
+def mitotip(ht: hl.Table, **_: Any) -> hl.Expression:
```

Comment:
Maybe `mitotip_id`?

Reply:
yes, we should be consistant about using `id` for all enumerated fields

Reply:
I think... I was actually wrong here.  `mitotip` is the data source name, and the actual mapped field is embedded as `trna_prediction_id`.  This aligns as is to what we've been doing elsewhere.

Reply:
is there an updated pipeline output with this fixed annotation?

Reply:
No, not yet.  I just updated the existing one because it'd take me much less time:

```
In [10]: ht = hl.read_table('gs://seqr-datasets/v03/GRCh38/MITO/runs/manual__2023-07-25T02-54-17.804650+00-00/annotations.ht')

In [11]: ht.globals.enums.mitotip
Out[11]: <StructExpression of type struct{trna_prediction: array<str>}>
```